website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroy Infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Delete Workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all infrastructure managed by a workspace.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. It's recommended that you destroy the infrastructure managed by a workspace prior to deleting the workspace itself, otherwise the infrastructure resources will continue to exist but will become unmanaged.
+You will need to go into your infrastructure providers to delete the resources manually.
+
+Destroy plans
+
+Before queuing a destroy plan, you must enable the `Allow destroy plans` setting.
+
+### Manually Destroy
+
+Adds a destroy plan to the run queue.
+
+### Automatically Destroy
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
+<!-- END: TFC:only name:pnp-callout -->
+
+Schedules a destroy planx to be run at some point in the future.
+
+This value will be automatically unset after an auto destroy run has been queued.
+
+-> **Note:** Automatic destroy plans will not prompt for apply approval in the Terraform Cloud web UI.
```

Comment:
I don't think this needs to be a callout? Unless this is conveying something the user needs to know? Is apply approval required in other ways of using this?
```suggestion
Automatic destroy plans do not prompt you for apply approval in the Terraform Cloud user interface.
```

Reply:
Updated to include your suggested verbiage, but I still left it as a callout for now. I feel like it's somewhat important for this to be a callout because auto destroy runs work in a different way from all other types of runs currently which could be potentially dangerous if users expect that auto destroys will still respect the current run approval process. But definitely flexible here if you don't think it should be a callout.

Reply:
Hmm I think adding a little bit of the context you described above would be helpful to inform folks _why_ this important. Can you add a recommendation to this informing folks why they should be wary of using this. Maybe something like:

```suggestion
-> **Note:**  Unlike other runs, automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend thoroughly reviewing your automatic destroy plans for accuracy.
```

Reply:
Update with this, does it seem sufficient?

```
!> **Note:** Unlike other runs, automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend only using this setting for development environments.
```

Reply:
Oh that's **perfect**! Thank you for iterating on that with me!